examples/webextension/background.js

```diff
@@ -0,0 +1,15 @@
+const TRACKING_ID = 'UA-XXXXXXXX-YY';
+
+const { sendEvent } = new Metrics({
```

Comment:
I'm not 100% familiar w/ web extension background scripts, but do we need to import `Metrics` class/object before using it here? Or do we automatically get it in-scope since it's defined in the manifest.json's `background.scripts[]` array?